cloudflare-ddns

```diff
@@ -117,7 +117,7 @@ def update_dns_record(auth, zone_id, record, ip_address):
     update_resp = requests.put(
         CLOUDFLARE_ZONE_DNS_RECORDS_UPDATE_API.format(zone_id=zone_id, dns_record_id=record['id']),
         headers=dict(list(auth.items()) + [('Content-Type', 'application/json')]),
-        data=json.dumps({'type': record['type'], 'name': record['name'], 'content': ip_address}),
+        data=json.dumps({'type': record['type'], 'name': record['name'], 'content': ip_address, 'proxied':'true'}),
```

Comment:
You should use `True`, the boolean primitive, instead of the string `'true'`.

Additionally, this seems to be a breaking change, as this will end up enabling Cloudflare proxy for records where it was not previously enabled. Could you put this behavior behind a runtime flag?